nixos/tests/netbird.nix

```diff
@@ -12,6 +12,7 @@ import ./make-test-python.nix ({ pkgs, lib, ... }:
     };
   };
 
+  # TODO: confirm DynamicUser is working if/after netbird server is implemented
```

Comment:
There is no netbird server in this PR

Reply:
yeah, but it is very much relevant for the test, there is no confirmation the client works at all until we do have a server running.